### PR TITLE
Firstaid crop

### PR DESCRIPTION
**Pull request by u-fischer**
This resolves two problems of the crop package. See 

https://github.com/rrthomas/crop/issues/2
https://github.com/rrthomas/crop/issues/3

# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [n/a] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: required/firstaid/config-pdf.lua [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Tests needing \pdfoutput=1
+
+stdengine    = "pdftex"
```
Comment (josephwright) [inline required/firstaid/config-pdf.lua:3]: Up-to-date `l3build` doesn't need `strengine` as it uses the first entry in `checkengines` (it does no harm, just for future ref.)

File: required/firstaid/latex2e-first-aid-for-external-files.dtx [hunk 2]
```diff
@@ -443,6 +443,41 @@
 }
 %    \end{macrocode}
 %
+% \subsection{The \pkg{crop} package first aid}
+%
+%    The crop packages fails currently due to two LaTeX changes:
```
Comment (josephwright) [inline required/firstaid/latex2e-first-aid-for-external-files.dtx:448]: `\LaTeX{}`

**Review by josephwright — APPROVED** [review]

File: required/firstaid/testfiles-pdf/firstaid-crop-mirror.lvt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+\documentclass{article}
+\input{regression-test}
+\usepackage[mirror]{crop}
+\begin{document}
+\START
+Some text
+\newpage
+more text
+\end{document}
```
Comment (FrankMittelbach) [inline required/firstaid/testfiles-pdf/firstaid-crop-mirror.lvt:9]: newline missing (not important but shows in the diff

File: required/firstaid/testfiles-pdf/firstaid-crop-stockwidth.pvt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+\usepackage[center]{crop}
+
+
+\begin{document}
+\START
+\showoutput
+abc
+\newpage
+abc
+\end{document}
```
Comment (FrankMittelbach) [inline required/firstaid/testfiles-pdf/firstaid-crop-stockwidth.pvt:12]: ditto

**Review by FrankMittelbach — APPROVED** [review]

**Pull request merged**